src/kOS/Screen/ConnectivityInterpreter.cs

```diff
@@ -23,19 +23,32 @@ public class ConnectivityInterpreter : Interpreter, IUpdateObserver
         public ConnectivityInterpreter(SharedObjects shared) : base(shared)
         {
             Shared.UpdateHandler.AddObserver(this);
-            CreateProgressBarSubBuffer();
+            CreateProgressBarSubBuffer(this);
+            AddResizeNotifier(CreateProgressBarSubBuffer);
         }
 
-        private void CreateProgressBarSubBuffer()
+        ~ConnectivityInterpreter()
         {
-            progressBarSubBuffer = new SubBuffer();
+            RemoveResizeNotifier(CreateProgressBarSubBuffer);
+        }
```

Comment:
I just realized this won't work.  The destructor never gets called until after you orphan this instance.  This instance can't orphan until after you call ``RemoveResizeNotifier()``, so this can never actually get called.  This has to be moved elsewhere in the logic, not in the destructor.